packages/cms-lib/projects.js

```diff
@@ -175,8 +176,37 @@ async function createProject(dest, type, repoName, sourceDir, options = {}) {
   return success;
 }
 
+/**
+ * Writes a copy of the boilerplate project to dest.
+ * @param {String} dest - Dir to write project src to.
+ * @param {String} type - Type of project to create.
+ * @param {String} repoName - Name of GitHub repository to clone.
+ * @param {String} sourceDir - Directory in project that should get copied.
+ * @param {Object} options
+ * @returns {Boolean} `true` if successful, `false` otherwise.
+ */
+async function createVueProject(dest, type, repoName) {
+  const relativePath = path.relative(process.cwd(), dest);
+
+  return new Promise(resolve => {
+    const vueInit = spawn(
+      'vue',
+      ['init', `HubSpot/${repoName}`, relativePath],
+      {
+        stdio: 'inherit',
+        shell: false,
+      }
+    );
+
+    vueInit.on('exit', () => {
+      resolve();
+    });
+  });
+}
```

Comment:
One thing that I think is unclear here is that `hs` is running `vue` under the hood. I am not 100% sure that this adds a ton of value since the user has to have the vue CLI installed. If it does add value, I think we need to make it as clear as we can that the `vue` CLI is be run by the `hs` because if there is a problem it needs to be clear that the problem is with `vue` not `hs`.